### ISSUE TITLE
Could not find a declaration file for module 'simplebar'

### ISSUE THREAD

**Issue by kamiljaworskiicf** [Grsmto/simplebar#660]
**Describe the bug**
When importing SimpleBar from 'simplebar', TS shows bellow error:
TS7016: Could not find a declaration file for module 'simplebar'. '/path' implicitly has an 'any' type.
Try `npm i --save-dev @types/simplebar` if it exists or add a new declaration (.d.ts) file containing `declare module 'simplebar';` 

**To Reproduce**
1. Create new project with 'npm create vite@latest'
2. Select Vanilla/Typescript environment 
3. npm i simplebar
4. Add import to the main.ts => import SimpleBar from 'simplebar'

**Expected behavior**
Typescript types should be included within the simplebar folder. 
@types/simplebar are depreciated by now, because package includes types in the simplebar-core. 

| Software         | Version(s) |
| ---------------- | ---------- |
| SimpleBar        | 6.2.2
| npm/Yarn         | 8.11.0

**Comment by RFCarreira33** [comment]
If you're using React try `npm install simplebar-react --save`  and then import 
```js 
import SimpleBar from 'simplebar-react';
import 'simplebar-react/dist/simplebar.min.css';
```
https://github.com/Grsmto/simplebar/tree/master/packages/simplebar-react#readme

**Comment by Grsmto** [comment]
Fix published `simplebar@6.2.3`.
